managed-signers/src/app/api/signer/route.ts

```diff
@@ -0,0 +1,43 @@
+import neynarClient from "@/lib/neynarClient";
+import { generate_signature } from "@/utils/generateSignature";
+import { getFid } from "@/utils/getFid";
+import { NextResponse } from "next/server";
+
+export async function POST() {
+  try {
+    const createSigner = await neynarClient.createSigner();
```

Comment:
small nit, I'd make a util function that does the create signer and register signed key in one function, and returns the signed key

Reply:
good point